src/containers/Secrets/Secrets.js

```diff
@@ -0,0 +1,149 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { InlineNotification } from 'carbon-components-react';
+import Modal from '../SecretsModal';
+import DeleteModal from '../../components/SecretsDeleteModal';
+import Table from '../../components/SecretsTable';
+import {
+  clearNotification,
+  deleteSecret,
+  fetchSecrets
+} from '../../actions/secrets';
+import {
+  getSecrets,
+  getSecretsErrorMessage,
+  isFetchingSecrets
+} from '../../reducers';
+
+export /* istanbul ignore next */ class Secrets extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      openNewSecret: false,
+      openDeleteSecret: false,
+      toBeDeleted: ''
```

Comment:
this is used as an object later in the code, so the default value should be either `null` or `{}` to avoid confusion